Makefile

```diff
@@ -279,22 +281,11 @@ $(SECURITY_GO_BINS): depend
 .PHONY: build
 build: $(PILOT_GO_BINS) $(MIXER_GO_BINS) $(SECURITY_GO_BINS)
 
-# The following are convenience aliases for most of the go targets
-# The first block is for aliases that are the same as the actual binary,
-# while the ones that follow need slight adjustments to their names.
-
-IDENTITY_ALIAS_LIST:=istioctl mixc mixs pilot-agent servicegraph sidecar-injector
-.PHONY: $(IDENTITY_ALIAS_LIST)
-$(foreach ITEM,$(IDENTITY_ALIAS_LIST),$(eval $(ITEM): ${ISTIO_OUT}/$(ITEM)))
-
-.PHONY: istio-ca
-istio-ca: ${ISTIO_OUT}/istio_ca
-
-.PHONY: node-agent
-node-agent: ${ISTIO_OUT}/node_agent
-
-.PHONY: pilot
-pilot: ${ISTIO_OUT}/pilot-discovery
+# The following are convenience aliases for most of the Go targets
+MAPPING_MODULES=pilot.pilot-discovery proxy.pilot-agent istioctl.istioctl sidecar_initializer.sidecar-initializer mixs.mixs mixc.mixc servicegraph.servicegraph node-agent.node_agent istio-ca.istio_ca
```

Comment:
Most of my earlier comments about the mapping still apply (most of which are points for an approver to consider), though a definite need is to update "sidecar_initializer.sidecar-initializer" to reflect the new sidecar-injector name.

Reply:
@mattdelco Thanks for catching it. Somehow I missed the name change to sidecar-injector.

I think that the best thing for me to do here is to backoff these changes since it's not related to the issue that the PR is trying to address anyway